### DISCUSSION TITLE
HELLO VerticaPy community!

### DISCUSSION THREAD

**Discussion by mail4umar** [vertica/VerticaPy#474] [Announcements]
**HELLO VerticaPy community!** 

We are excited to kick off this section with a couple cool announcements! 📢 

1️⃣ First up, we are revamping the **Discussions tab** as our way of updating and interacting with the VerticaPy community 🗣️. Here's our current plan for the different sections: 

🎉 **Announcements** : Important updates will be shared in this section. This will create a historical record that summarizes the modifications to the code. 

👋 **General** : Minor modifications to the code or less important communications will be announced here. 

💡 **Ideas** : We will use the Ideas button to take suggestions about new specs. Users can provide ideas for new features that benefit their use cases. The developers can then use these ideas to create polls to prioritize development. 

👍 **Polls** : We plan to use user polls to gauge the community's interest for different features. These will give us a more transparent view of the customer demand.

❗ **Potential bug**: When something is not working right, it can be reported and discussed here. We call it a potential bug because it might be an unexpected behavior caused by a usage mistake.

🙋‍♂️ **Q&A** : We would like the community to request help in the Q&A section instead of creating issues. We want to save the issues tab for bugs inside the code. So if you are stuck or need help, please feel free to write your problem in the Q&A section. 

For the **Issues** tab ❌, we will create new **labels** for each independent portion of the code (e.g. plotting). This way every developer will be responsible for fixing the bugs pertaining to their own particular specialties. Every folder inside the Verticapy/verticapy directory will have its own label: **CONNECT** , **CORE**, **DATASETS** etc. 

Lastly, **milestone** 🎯 will become a subtask and will no longer be linked to the version. Each Project will be composed of the different to-do tasks ✅, allowing for multiple milestones. This lets us see a milestone as a *sprint* and the project as *Program increment* (PI). 

For more details on contributing, please refer to [contributing help](https://github.com/vertica/VerticaPy/blob/master/CONTRIBUTING.md). (Note that this link will be updated to reflect the latest changes soon) 

2️⃣ We have also restructured the VerticaPy code to make it more *efficient* and *intelligible*. To achieve this, we have created parent folders for different areas of focus in the code (e.g. machine learning, sql, etc). The code for each area of focus will be placed underneath the appropriate parent folder. 

This new organization should improve both code readability and ease of collaboration.

**Comment by oualib** [comment]
I think that's the way to go to bring VerticaPy to the next level. If everything goes well, we'll reach the v1.0.0 this year.

**Comment by thepranj** [comment]
Nice! Excited for what's to come...
